theme/banner/_print.scss

```diff
@@ -1,11 +1,25 @@
 /*
  Banner (print view)
  */
+%theme-wet-boew-banner-print-brightness-0 {
+	img {
+		filter: brightness(0);
```

Comment:
Is that fix is needed every time a SVG is used, or it is unique to wet-boew theme?

Reply:
It's needed for SVGs that contain inline media queries that change their colours to white in screen viewports. It also benefits their fallback PNGs. WET's theme needs it.

I'm currently looking into porting this PR to other affected WET themes. I'm taking the behaviour of the inline media queries in the other themes's SVG images into account. So similar kinds of CSS filters might be present in some of my port PRs.

Reply:
Do you think the scope of that CSS should be made broader than only for some of the wet-boew theme elements.

Reply:
I'm still going through the other themes, but based on what I've seen so far, I think this kind of thing would work best on a theme-by-theme basis. Some themes use SVGs with other kinds of media queries that'll need to be handled differently.

Reply:
ok, then this fix is only for our WET logo to be printable. Thanks @EricDunsworth for the clarification.